### PR TITLE
CLI workload Subcommand

### PR DESCRIPTION
**Pull request by jordan-rash**
The `Run` functions are not correct as I do not believe the structs for creating a workload exist yet.  Currently agent structs are placeholders to ensure all the required fields are represented 

This will need to be rebased once #384 is merged - putting in draft until then

### CODE REVIEW

File: cmd/nex/workload.go [hunk 1]
```diff
@@ -1,10 +1,207 @@
...
+	Name             string         `placeholder:"myworkload" help:"Name of the workload"`
+	WorkloadType     string         `name:"type" default:"native" help:"Type of workload"`
+	Description      string         `help:"Description of the workload"`
+	Argv             []string       `placeholder:"--flag,value,-a,..." default:"" help:"Arguments to pass to the workload"`
+	Env              map[string]any `placeholder:"KEY=VALUE;KEY2=VALUE2;..." default:"" help:"Environment variables to pass to the workload"`
+	TriggerSubjects  []string       `placeholder:"subject1,subject2" default:"" help:"Trigger subjects to register for subsequent workload execution, if supported by the workload type"`
+	HostServicesURL  string         `placeholder:"nats://localhost:4222" default:"" help:"NATS URL to host services"`
+	HostServicesJWT  string         `placeholder:"etahsy..." default:"" help:"NATS Authentication JWT for host services"`
+	HostServicesSeed string         `placeholder:"SUA..." default:"" help:"NATS Authentication Seed for host services"`
+	Devrun           bool           `help:"Use default values for workload; chooses node at random"`
```
Comment (kthomas) [inline cmd/nex/workload.go:48]: Are we keeping `devrun`?

Comment (kthomas) [reply cmd/nex/workload.go:48]: The `run` command should use sane defaults by... default.

File: cmd/nex/workload.go [hunk 2]
```diff
@@ -1,10 +1,207 @@
...
+}
+
+type StopWorkload struct {
+	WorkloadId string `arg:"" required:"" help:"ID of workload to stop"`
+	TargetId   string `help:"Node to send stop workload on"`
+	IssuerKey  string `default:"${defaultConfigPath}/issuer.nk" help:"Issuer key file to use for workload"`
+	Reason     string `default:"" help:"Reason for stopping workload"`
+	Immediate  bool   `default:"false" help:"Stop workload immediately; workload will try and gracefully stop otherwise"`
+
+	Devrun bool `help:"Use default values for workload; chooses node at random"`
```
Comment (kthomas) [inline cmd/nex/workload.go:177]: Preference would be to default to sane values.

File: cmd/nex/workload_test.go [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+func TestStartWorkload(t *testing.T) {
+	confDir, err := initWorkloadCommand(t)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	w := Workload{
+		Run: RunWorkload{
+			Devrun:       false,
```
Comment (kthomas) [inline cmd/nex/workload_test.go:44]: We should remove this flag and have tests asserting all the things that need defaults are set if they aren't provided.

**Comment by jordan-rash** [comment]
Completed in #414

**Pull request closed without merging**